examples/datasets/_01_datasets_basics.py

```diff
@@ -119,26 +119,46 @@ def create_index(self):
 # %%
 # If we now iterate the dataset, it will iterate over the unique groups.
 #
-# Grouping also changes the meaning of a "single datapoint".
+# Grouping also changes the meaning of a "single datapoint". # TODO how do we change this explanation?
```

Comment:
Probably this should not be referred to as single datapoint since there is now the datapoint_label property and a is_single_datapoint method.
How should we call it instead?
Maybe something like "single data unit"?

Reply:
Can you remove the todo here?

Reply:
The TODO is not important anymore right?